src/app/dashboard/page.tsx

```
@@ -1,51 +1,272 @@
+      (sum, c) => sum + (c.donors ?? 0),
+      0,
+    );
+    const ongoingCampaigns = sortedCampaigns.filter(
+      (c) => c.status === "published",
+    ).length;
+    const totalRaised = sortedCampaigns.reduce(
+      (sum, c) => sum + (c.raised ?? 0),
+      0,
+    );
+    return { donationsReceived, ongoingCampaigns, totalRaised };
+  }, [competitionCampaigns]);
```

ESLint

[error] Compilation Skipped: Existing memoization could not be preserved

React Compiler has skipped optimizing this component because the existing manual memoization could not be preserved. The inferred dependencies did not match the manually specified dependencies, which could cause the value to change more or less frequently than expected. The inferred dependency was `sortedCampaigns`, but the source dependencies were [competitionCampaigns]. Inferred different dependency than source.

/home/jailuser/git/src/app/dashboard/page.tsx:117:25
  115 |   }, [competitionCampaigns, sortKey]);
  116 |
> 117 |   const stats = useMemo(() => {
      |                         ^^^^^^^
> 118 |     const donationsReceived = sortedCampaigns.reduce(
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 119 |       (sum, c) => sum + (c.donors ?? 0),
      …
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 129 |     return { donationsReceived, ongoingCampaigns, totalRaised };
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 130 |   }, [competitionCampaigns]);
      | ^^^^ Could not preserve existing manual memoization
  131 |
  132 |   const topCampaigns = useMemo(() => sortedCampaigns.slice(0, 6), [sortedCampaigns]);
  133 |

(react-hooks/preserve-manual-memoization)